src/index.html

```diff
@@ -21,10 +21,12 @@ <h1>To Do List</h1>
                     <h2>Today's To Do</h2>
                     <i class="fas fa-sync sync-icon"></i>
                 </div>
-                <div class="task-name">
+                <form id ="task-name" action="POST" class="input-icon">
                     <input type="text" id="description" class="task-desk" placeholder="Add to your list...">
-                    <i class="fas fa-plus-square plus-icon"></i>
-                </div>
+                    <button id="taskSubmit" type="submit">
```

Comment:
- [ ] Please fix the submitting form error to add new todos, after I try to submit new todo, it gives me `Cannot GET /POST` error
![Screenshot (1)](https://user-images.githubusercontent.com/47625407/193023873-607b8a7a-52c1-4872-b003-4634678c18f4.png)